src/current/cockroachcloud/migrations-page.md

```diff
@@ -102,14 +102,10 @@ The dump file must be smaller than 4 MB. `INSERT` and `COPY` statements will be
 <section class="filter-content" markdown="1" data-scope="postgres mysql">
 ### Use Credentials
 
-{{site.data.alerts.callout_info}}
-{% include feature-phases/preview.md %}
-{{site.data.alerts.end}}
-
-The Schema Conversion Tool can connect directly to a PostgreSQL or MySQL database to obtain the schema. To add a schema using credentials:
+The MOLT Schema Conversion Tool can connect directly to a PostgreSQL or MySQL database to obtain the schema. To add a schema using credentials:
```

Comment:
If I'm not mistaken, we also support Oracle and SQL server, but those have the `beta` label. @Jeremyyang920 should we mention that we support it in preview?

Reply:
I didn't realize. I can address this in a future PR. A DOC issue would be appreciated when you get a chance! I don't think I've seen one.

Reply:
Not for oracle and sql server for credentials. We only did it for pg and mysql.